crates/common/src/bid_submission/submission.rs

```diff
@@ -353,4 +363,8 @@ pub struct SignedBidSubmissionDeneb {
     pub execution_payload: ExecutionPayload,
     pub blobs_bundle: BlobsBundle,
     pub signature: BlsSignature,
+    /// The Merkle proofs of inclusion as needed by the Constraints API.
+    /// Reference: <https://docs.boltprotocol.xyz/api/builder#get_header_with_proofs>
```

Comment:
update to https://docs.boltprotocol.xyz/technical-docs/api/builder#get_header_with_proofs